dev-resources/template/2_lrs.yml

```diff
@@ -1,44 +1,21 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Description: "SQL LRS and DB Init Script"
 Parameters:
-  # Networking
   VPCId:
-    Description: "VPC on which to run SQL LRS"
+    Description: VPC on which to run SQL LRS
     Type: AWS::EC2::VPC::Id
   DBSubnets:
     Description: Subnets on which the DB is running
     Type: List<AWS::EC2::Subnet::Id>
   # DB Details
-  DBName:
-    Description: PG Database name. Ignored if DBSnapshotIdentifier is provided
-    Type: String
-    Default: "lrsql_demo"
-  DBHost:
-    Description: Write hostname of PG db
-    Type: String
-  DBPort:
-    Description: Port of PG db
-    Type: String
-    Default: 3306
-  DBInstanceSG:
-    Type: AWS::EC2::SecurityGroup::Id
-    Description: DB instance security group
-  DBMasterUserName:
-    Description: Name of admin user
+  DBStackName:
+    Description: DB Stack Reference
     Type: String
     Default: yetadmin
```

Comment:
you left copypasta in the default. IMO `DBStackName` should not have a default at all

Reply:
addressed via https://github.com/yetanalytics/lrsql/pull/276/commits/e0d190ce91168c22c1d99a61206f14b3da2646d8